### PR TITLE
fix date format

### LINKED ISSUE #475: Edit entity is broken
It doesn't load the data, so the edit entity is broken

### PR DESCRIPTION
**Pull request by nonomoho**
date-fns library has a breaking change in the last release :
https://date-fns.org/v2.0.0-beta.2/docs/parseISO#v2.0.0-breaking-changes

Fix #475

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-vuejs/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs [hunk 1]
```diff
@@ -24,6 +24,19 @@ import { shallowMount, createLocalVue, Wrapper } from '@vue/test-utils';
...
 import Router from 'vue-router';
 
+<%_
+    let include = false;
+    for (idx in fields) {
+    const fieldName = fields[ idx ].fieldName;
+    const fieldType = fields[idx].fieldType;
+    if ([ 'Instant', 'ZonedDateTime' ].includes(fieldType) && !include) {
+        include = true;
+        _%>
```
Comment (pascalgrimaud) [inline generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs]: remove tab / space -> format

File: generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs [hunk 2]
```diff
@@ -95,6 +108,32 @@ describe('Component Tests', () => {
...
     });
 
+    <%_
+    let included = false;
+    for (idx in fields) {
+    const fieldName = fields[ idx ].fieldName;
+    const fieldType = fields[idx].fieldType;
+    if ([ 'Instant', 'ZonedDateTime' ].includes(fieldType) && !included) {
+        included = true;
+        _%>
```
Comment (pascalgrimaud) [inline generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs]: align -> format `_%>`

File: generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs [hunk 3]
```diff
@@ -24,6 +24,19 @@ import { shallowMount, createLocalVue, Wrapper } from '@vue/test-utils';
 import sinon, { SinonStubbedInstance } from 'sinon';
 import Router from 'vue-router';
 
+<%_
+    let include = false;
+    for (idx in fields) {
+    const fieldName = fields[ idx ].fieldName;
```
Comment (hdurix) [inline generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs]: Please reformat (`const ...` inside `for` block)

File: generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs [hunk 4]
```diff
@@ -24,6 +24,19 @@ import { shallowMount, createLocalVue, Wrapper } from '@vue/test-utils';
 import sinon, { SinonStubbedInstance } from 'sinon';
 import Router from 'vue-router';
 
+<%_
+    let include = false;
```
Comment (hdurix) [inline generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs]: Maybe a more explicit name

File: generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs [hunk 5]
```diff
@@ -95,6 +108,32 @@ describe('Component Tests', () => {
       comp = wrapper.vm;
     });
 
+    <%_
+      let included = false;
+      for (idx in fields) {
+      const fieldName = fields[ idx ].fieldName;
```
Comment (hdurix) [inline generators/entity-client/templates/vue/src/test/javascript/spec/app/entities/entity-management-update.component.spec.ts.ejs]: Please reformat

**Comment by nonomoho** [comment]
Bounty claimed: https://opencollective.com/generator-jhipster/expenses/10842

**Comment by pascalgrimaud** [comment]
just approuved

**Pull request merged**